runbotics/runbotics-desktop/src/action/file/file.action-handler.ts

```diff
@@ -97,6 +104,21 @@ export default class FileActionHandler extends StatelessActionHandler {
         return 'File saved successfully';
     }
 
+    async existsFile(input: FileExistsActionInput) {
+
+        if (!input.name) {
+            throw new Error('Name is not provided.');
+        }
+
+        if (input.path && !fs.existsSync(input.path)) {
+            throw new Error('Provided path does not exist.');
+        }
+
+        const filePath = path.join(input.path, input.name);
```

Comment:
does it make sense to make path optional? is there a use case when the path is not required to complete action succesfully?